src/liballoc_system/lib.rs

```diff
@@ -19,7 +20,9 @@
 #![feature(alloc)]
 #![feature(core_intrinsics)]
 #![feature(staged_api)]
+#![feature(custom_attribute)]
 #![cfg_attr(any(unix, target_os = "redox"), feature(libc))]
+#![alloc_kind = "lib"]
```

Comment:
Could this perhaps be called "rustc_alloc_kind" to clearly deliniate that it's rustc-specific?